tests/smoke_test.py

```diff
@@ -16,24 +15,6 @@ def mac_supported_iast_version():
 
 
 if __name__ == "__main__":
-    # ASM IAST smoke test
-    if (3, 6, 0) <= sys.version_info < (3, 12) and system() != "Windows" and mac_supported_iast_version():
-        # ASM IAST import error test
```

Comment:
Sorry I think I just lack context... But why are we removing this? Could we keep this if we bumped `(3,6,0)` to `(3,7,0)` and removed the ImportError check?